### PR TITLE
docs: Change license of Mangrove core to Business Source License 1.1

### PR DESCRIPTION
**Pull request by Louis-Amas**

### CODE REVIEW

**Review by espendk — COMMENTED** [review]
It seems some of the files that should be covered by BUSL are missing:

billede

**Review by espendk — CHANGES_REQUESTED** [review]
The LICENSE file should still contain all the other licenses (which we still refer to in other files), the BUSL license should just be added (instead of replacing everything).

**Comment by Louis-Amas** [comment]
> It seems some of the files that should be covered by BUSL are missing:
> 
> billede

I used the one you updated below in the Asana task
billede

**Comment by espendk** [comment]
> > It seems some of the files that should be covered by BUSL are missing:
> > billede
> 
> I used the one you updated below in the Asana task billede

which lists the exact same files as your list ;-)

File: src/AbstractMangrove.sol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier:	AGPL-3.0
+// SPDX-License-Identifier:	BUSL-1.1
```
Comment (espendk) [inline src/AbstractMangrove.sol]:
```suggestion
// SPDX-License-Identifier: BUSL-1.1
```

File: LICENSE
Comment (espendk) [inline LICENSE:1]: I'll push a revision of this.

**Review by espendk — CHANGES_REQUESTED** [review]

**Review by espendk — APPROVED** [review]
Phil has reviewed and approves.

**Pull request merged**